JavaScript/Topics/13.DOM_in_javascript/README.md

```diff
@@ -0,0 +1,155 @@
+
+
+# JavaScript HTML DOM
+
+
+
+## What is DOM ?
+ DOM stands for Document Object Model . It's an API (application programming interface) for manipulating HTML and XML documents. The DOM represents a document as a tree of nodes . It allows programs and scripts to dynamically access and update the content, structure, and style of a document.
+
+
+Below image shows a basic heirarchy in DOM :
+
+
+![DOM ](https://searchengineland.com/figz/wp-content/seloads/2015/05/Microsoft-PowerPoint.jpg)
+
+
+## Use of DOM in JavaScript
+<br>
+With the DOM , javascript get all the power it needs to create dynamic webpage . With DOM :
+
+* JavaScript can change all the HTML elements in the page
+
+* JavaScript can change all the HTML attributes in the page
+
+* JavaScript can change all the CSS styles in the page
+
+* JavaScript can remove existing HTML elements and attributes 
+
+* JavaScript can add new HTML elements and attributes
+
+* JavaScript can react to all existing HTML events in the page
+
+* JavaScript can create new HTML events in the page
+
```

Comment:
Write like this, without leaving extra lines

```
- JavaScript can change all the HTML elements in the page
- JavaScript can change all the HTML attributes in the page
- JavaScript can change all the CSS styles in the page
- JavaScript can remove existing HTML elements and attributes 
- JavaScript can add new HTML elements and attributes
- JavaScript can react to all existing HTML events in the page
- JavaScript can create new HTML events in the page
```

This will show up like the same only, less empty lines will be there... (Preview of above code)
- JavaScript can change all the HTML elements in the page
- JavaScript can change all the HTML attributes in the page
- JavaScript can change all the CSS styles in the page
- JavaScript can remove existing HTML elements and attributes 
- JavaScript can add new HTML elements and attributes
- JavaScript can react to all existing HTML events in the page
- JavaScript can create new HTML events in the page